src/main/java/no/ndla/taxonomy/rest/v1/Nodes.java

```diff
@@ -56,7 +56,7 @@ public Nodes(NodeRepository nodeRepository, NodeConnectionRepository nodeConnect
     @GetMapping
     @Operation(summary = "Gets all nodes")
     public List<EntityWithPathDTO> getAll(
-            @Parameter(description = "Filter by nodeType") @RequestParam(value = "nodeType", required = false) Optional<NodeType> nodeType,
+            @Parameter(description = "Filter by nodeType, could be a comma separated list :^)") @RequestParam(value = "nodeType", required = false) List<NodeType> nodeType,
```

Comment:
Du bruker :^ en del …

Reply:
Haha, har sett for mye på serenityos folka 😄 
https://happy-serenityos.linus.dev/